package.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Why is this in the root of the project?

Reply:
Should I migrate the project to npm workspace then?

Reply:
Let's discuss all of the changes in discord :)